kitsune/sumo/static/sumo/js/experiment-fxa-cta-topbar.js

```diff
@@ -0,0 +1,58 @@
+(function(Mozilla) {
+  'use strict';
+
+  // callback for UITour conditional
+  var rogerThat = function(sync) {
+
+    // only run if not syncing
+    var notSyncing = !sync.detail.data.setup;
+
+    if (notSyncing) {
+
+      var showPromotion = function(variation) {
+        document.addEventListener('DOMContentLoaded', function() {
+          var promotions = document.getElementById('promotions');
+          var showSpan = promotions.querySelector('span[data-variation="'+variation+'"]');
```

Comment:
Not sure if it matters, but I believe this will error out in IE <= 7.

Reply:
I assert this will never happen in any non-Firefox browser due to useragent conditional on https://github.com/mozilla/kitsune/pull/3099/files/1b22377aa69cf8a5b34ce3b3e570a84e80ef0cfb#diff-a4c6185ec7b16331560f1e94348ee1a7R45.